insightfinder/README.md

```diff
@@ -0,0 +1,30 @@
+# InsightFinder
+
+## Overview
+
+[InsightFinder][1] AIOps identifies system and application issues before they impact users. Powered by unsupervised machine learning, InsightFinder continuously learns from events, logs, metrics, and changes to detect anomalies, predict incidents, and remediate outages.
+
+InsightFinder ingests data from Datadog through standard APIs and finds anomalous events before your business is impacted.
+
+## Setup
+
+### Installation
+
+To configure the integration and send data to InsightFinder, see the [InsightFinder-Datadog Integration][2]. You need a Datadog [API key and application key][5].
+
+If you do not have an InsightFinder account, [sign up][3] to start your free trial.
+
+## Support
+
+Contact [Datadog support][6] or email [InsightFinder support][4].
+
+## Further Reading
+
+Additional helpful documentation, links, and articles:
```

Comment:
```suggestion
```

missed this the first time around! there aren't any further reading links on this page, so let's get rid of this section.